tests/options.js

```diff
@@ -38,36 +38,30 @@ define(function () {
                     ]
                 }
             ],
-        },
+        }
+    ];
+
+    var fixtures = [
         {
             name: "Some Fixture",
-            sourceUri: "jr://fixture/some-fixture",
-            defaultId: "somefixture",
-            rootNodeName: "foos",
-            levels: [
-                {
-                    nodeName: "foo",
-                    subsets: [
-                        {
-                            name: "woos",
-                            // should handle quotes
-                            selector: "@foo_type=\"woo\""
-                        }
-                    ]
-                },
-                {
-                    nodeName: "bar",
-                    subsets: [
-                        {
-                            name: "eggs",
-                            selector: "@bar_type='eggs'"
-                        }
-                    ]
-                }
+            sourceUri: "jr://fixture/item-list:some-fixture",
+            columns: [
+                'column1',
+                'column2',
+                'column3'
+            ]
```

Comment:
Yeah. this is why I decided to upload my progress here. This isn't done by any means. Just to make sure that I'm going along the right path